rest/data/project.go

```diff
@@ -50,6 +53,40 @@ func FindProjectById(id string, includeRepo bool, includeProjectConfig bool) (*m
 	return p, nil
 }
 
+// RequestAWSAccess creates a JIRA ticket that requests build to create an AWS user for the specified project.
+func RequestAWSAccess(projectIdentifier string) error {
```

Comment:
Just thinking about this more 😅 although we do intend to get rid of this for a more generalized solution, this is very hardcoded to mongoDB's specific workflow, and Evergreen _is_ open source. Can we have the Jira project be configurable on the admin page? And if the value isn't populated we can set the UI not to show the checkbox I think (this would also be good in case we end up having an issue with the feature).

Reply:
I'll also ask Matthew to create the ticket for the project to make this generalized so we can link it here.

Reply:
https://jira.mongodb.org/browse/PM-3212

Reply:
done and tested on staging